src/Microsoft.Diagnostics.Monitoring.WebApi/Controllers/DiagController.cs

```diff
@@ -75,10 +75,13 @@ public Task<ActionResult<IEnumerable<Models.ProcessIdentifier>>> GetProcesses()
                 {
                     defaultProcessInfo = await _diagnosticServices.GetProcessAsync(null, HttpContext.RequestAborted);
                 }
-                catch (Exception)
```

Comment:
Let's add the general Exception catch block at the end with a filter for non OperationCanceledException instances. Consider adding logging at warning level to the catch block to notify the user when a non-OperationCanceledException exception occurs.